portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,15 +67,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String firstName = request.getParameter("firstName");
     String lastName = request.getParameter("lastName");
     long timestamp = System.currentTimeMillis();
-    // TODO(aabundis): Verify that a comment was written, no empty submissions.
```

Comment:
Is this not needed anymore? What happens when an empty submission is written?

Reply:
That is validated using the html form attribute "required". So the form cannot be submitted if one of this fields remained blank.

Reply:
This is validated using the HTML Form attribute "required". Therefore the form cannot be submitted as long as a field is blank.

Reply:
Gotcha, thanks for clarifying!